action.yml

```diff
@@ -119,3 +129,34 @@ runs:
         bluebuild build -v --push ${RECIPE_PATH} \
           --registry ${{inputs.registry}} \
           --registry-namespace ${{inputs.registry_namespace}}
+
+    - name: Build Squashed Image
+      shell: bash
+      if: ${{ inputs.squash == 'true' }}
+      env:
+        COSIGN_PRIVATE_KEY: ${{ inputs.cosign_private_key }}
+        GH_TOKEN: ${{ inputs.registry_token }}
+        GH_PR_EVENT_NUMBER: ${{ inputs.pr_event_number }}
+        RECIPE: ${{ inputs.recipe }}
+        # Uses GitHubs ternary syntax to set cli version, see https://docs.github.com/en/actions/learn-github-actions/expressions#example
+        CLI_VERSION_TAG: ${{ inputs.use_unstable_cli == 'true' && 'main' || 'v0.8.4' }}
```

Comment:
This should be made into a global variable and passed into both places where it's needed.

Reply:
Already taken care of